bin/clean_os_dupl.sh

```diff
@@ -3,18 +3,31 @@
 # Exit immediately if a command exits with a non-zero status
 set -e
 
-# Fix OS issue/bug: "sudo: unable to resolve host..."
```

Comment:
I've realized that it doesn't really make any difference, just an extra line. So removed to reduce script's length.